descriptor.md

```diff
@@ -61,10 +61,10 @@ Extended _Descriptor_ field additions proposed in other OCI specifications SHOUL
 
 The _digest_ property of a Descriptor acts as a content identifier, enabling [content addressability](http://en.wikipedia.org/wiki/Content-addressable_storage).
 It uniquely identifies content by taking a [collision-resistant hash](https://en.wikipedia.org/wiki/Cryptographic_hash_function) of the bytes.
-If the identifier can be communicated in a secure manner, one can retrieve the content from an insecure source, calculate the digest independently, and be certain that the correct content was obtained.
+If the digest can be communicated in a secure manner, one can retrieve the content from an insecure source, recalculate the digest independently, and be certain that the correct content was obtained.
 
-The value of the digest property, the _digest string_, is a serialized hash result, consisting of an _algorithm_ portion and a _hex_ portion.
-The algorithm identifies the methodology used to calculate the digest; the hex portion is the lowercase hex-encoded result of the hash.
+The value of the digest property, the _digest string_, is a serialized hash result, consisting of an _algorithm_ portion (the "algorithm identifier") and a _hex_ portion (the digest).
```

Comment:
I would prefer we refer to the whole string as the “digest” and the part after the semicolon as the “hash” or the “hex” to match our [definition][1] and [implementation][2].  Calling the hash/hex portion the “digest” if confusing if the whole thing is so similar (the “digest string”) and is represented by the `Digest` type.

[1]: https://github.com/opencontainers/image-spec/blame/37f2a1b20c1f59e9a7dcdbf7a9fe50144912960c/descriptor.md#L72-L74
[2]: https://github.com/opencontainers/go-digest/blob/v1.0.0-rc0/digest.go#L30-L36